README.md

```diff
@@ -1,237 +1,25 @@
-# gravity-ui-web
+# Gravity UI
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/buildit/gravity-ui-web.svg)](https://greenkeeper.io/)
```

Comment:
Can we keep the Greenkeeper badge please?

Reply:
Is greenkeeper working on the overall monorepo? The badge is still in the README.md for gravity-ui-web... https://github.com/buildit/gravity-ui-web/blob/feature/lerna-wrapper/packages/gravity-ui-web/README.md

Reply:
I _think_ so.

A quick glance at their docs confirms that they do support monorepos. But it looks like we might need to add a `greenkeeper.json` config file for it to work properly: https://greenkeeper.io/docs.html#monorepo

I'd suggest we gets this merged and see what happens. If Greenkeeper stops working, we can always add that config file later.

Reply:
I've added back the back to the root `README.md`. I'm suspicious Greenkeeper is only going to run against the root `package.json`, but hopefully I'm wrong.